modules/check_fastq.py

```diff
@@ -0,0 +1,91 @@
+from typing import Union
+
+
+def sort_by_length(seqs: dict, length_bounds: Union[int, tuple[int]] = (0.2 ** 32)):
+    """
+
+    Returns a list of sequence names with length out of the given range().
+    Allows to sort out unsuitable sequences based on their length.
+
+    Arguments:
+    -dictionary with keys -strings (names of sequences): values-tuples of strings (sequence, quality scores)
+    -length_bounds: int or tuple[int], int = the maximum allowed length, tuple = allowed range of lengths
+                    0.2**32 by default
+
+    """
+    unsuitable_by_length = []
+    for name, (sequence, quality) in seqs.items():
+        if type(length_bounds) is tuple:
+            if length_bounds[0] > len(sequence) > length_bounds[1]:
+                unsuitable_by_length.append(name)
+        elif type(length_bounds) is int:
+            if len(sequence) > length_bounds:
+                unsuitable_by_length.append(name)
```

Comment:
Я бы тут не разбивал это на две ооочень похожие ветки логики, а свел бы одну к другой

А еще для проверки типов принято использовать функцию isinstance

```suggestion
        if isinstance(length_bounds, int):
            length_bounds = (0, length_bounds)
        
        if length_bounds[0] > len(sequence) > length_bounds[1]:
            unsuitable_by_length.append(name)
```